op-node/rollup/types.go

```diff
@@ -489,6 +489,7 @@ func (c *Config) IsHoloceneActivationBlock(l2BlockTime uint64) bool {
 		!c.IsHolocene(l2BlockTime-c.BlockTime)
 }
 
+// TODO rename to IsIsthmusActivationBlock (this will require quite a bit of renaming)
```

Comment:
I think its fine to leave it `IsInteropActivationBlock` in this PR, at some point we will need to modify it. I think that using the final name in the smart contract + L1 attributes tx will create less churn